### PR TITLE
Added ability to specify customChromeDriverLocation

### PR DESCRIPTION
**Pull request by praveendvd**
This allows to run electron application testing using chrome driver service by pointing to the chrome driver installed using "npm install electron-chromedriver"

### CODE REVIEW

**Comment by praveendvd** [comment]
@christian-bromann  Hi i have added option to specify customchromedriver path. So that we can point it to chromedriver installed using electron-chromedriver

File: README.md [hunk 1]
```diff
@@ -111,6 +111,14 @@ Example: `wdio-chromedriver.log`
 
 Type: `string`
 
+### chromedriverCustomPath
+To use a custome chromedriver different than the one installed through "chromedriver npm module", provide the path. 
+
+Example: `./chromedriver.exe or d:/driver/chromedriver.exe`
```
Comment (christian-bromann) [inline README.md:117]:
```suggestion
Example: `/path/to/chromedriver` (Linux / MacOS), `./chromedriver.exe` or `d:/driver/chromedriver.exe` (Windows)
```

**Review by christian-bromann — COMMENTED** [review]
The change makes sense to me, can we add a test to check that it is actually using the custom chromedriver binary rather than the detected one?

**Comment by praveendvd** [comment]
Will do that thanks @christian-bromann  for looking into this

File: tests/launcher.test.js [hunk 2]
```diff
@@ -354,4 +354,40 @@ describe('ChromeDriverLauncher launcher', () => {
...
+            options.chromedriverCustomPath = undefined
+            const Launcher = new ChromeDriverLauncher(options, capabilities, config)
+            Launcher._redirectLogStream = jest.fn()        
+            await Launcher.onPrepare()        
+            expect(Launcher.chromedriverCustomPath).not.toBeUndefined
+        })
+    })
 })
+
+
```
Comment (christian-bromann) [inline tests/launcher.test.js:393]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -111,6 +111,11 @@ Example: `wdio-chromedriver.log`
...
-----
+### chromedriverCustomPath
+To use a custome chromedriver different than the one installed through "chromedriver npm module", provide the path. 
+
+Example: `/path/to/chromedriver` (Linux / MacOS), `./chromedriver.exe` or `d:/driver/chromedriver.exe` (Windows)
+
+Type: `string`
 
-For more information on WebdriverIO see the [homepage](http://webdriver.io).
+For more information on WebdriverIO see the [homepage](http://webdriver.io).
```
Comment (christian-bromann) [inline README.md:121]:
```suggestion
For more information on WebdriverIO see the [homepage](https://webdriver.io).
```

**Review by christian-bromann — APPROVED** [review]
Nice, this looks good to me 👍

**Comment by christian-bromann** [comment]
Feel free to merge and release with a `minor` version bump (just trigger this workflow: https://github.com/webdriverio-community/wdio-chromedriver-service/actions/workflows/release.yml)

**Comment by praveendvd** [comment]
@christian-bromann  I am not sure if i have the rights to do it , i can't see merge or run workflow options , could you let me know how to do it , i haven't done it before

**Comment by christian-bromann** [comment]
I just fixed the org structure, you should have all necessary access rights now. Please go ahead and Squash/merge this PR, then open the [release workflow](https://github.com/webdriverio-community/wdio-chromedriver-service/actions/workflows/release.yml), click on "Run workflow" and change the first input field from `patch` to `minor`. Let me if you have any questions.

**Pull request merged**